GVFS/GVFS.Installer.Mac/scripts/preinstall

```diff
@@ -17,3 +17,10 @@ if [ "$isKextLoaded" -gt 0 ]; then
     echo $unloadCmd
     eval $unloadCmd || exit 1
 fi
+
+GVFSBINPATH="/usr/local/vfsforgit/gvfs"
+if [ -f "${GVFSBINPATH}" ]; then
+    unmountCmd="${GVFSBINPATH} service --unmount-all"
+    echo $unmountCmd
+    eval $unmountCmd || exit 1
```

Comment:
Just to double check will the eval print any errors that come up from the call?  
This follows the format of the other commands and lgtme.

Reply:
yes it does.